File: src/codes/Users_Manual/stm_prop.rs
# use autd3::prelude::*;
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let stm = FociSTM {
    foci: vec![Point3::origin(), Point3::origin()],
    config: 1.0 * Hz,
};
dbg!(stm.sampling_config()?.freq()); // -> 2Hz
# Ok(())
# }

File: src/codes/Users_Manual/stm_prop.cpp
//~#include<iostream>
//~#include<autd3.hpp>
//~int main() {
//~using namespace autd3;
FociSTM stm(
    std::vector{
        Point3::origin(),
        Point3::origin(),
    },
    1.0f * Hz);
std::cout << stm.sampling_config().freq() << std::endl;  // -> 2Hz
                                                         //~return 0; }